tests/tree_xml_parser_appendices_tests.py

```diff
@@ -645,3 +645,18 @@ def test_process_separated_by_header(self):
         self.assertEqual(0, len(a2.children))
         self.assertEqual(['1111', 'A', 'a', 'p1'], amarkerless.label)
         self.assertEqual(0, len(amarkerless.children))
+
+    def test_appendix_letter_dash(self):
```

Comment:
Like the change in regparser/grammar/atomic.py, this is done in the interpretation headers tests now. So this could be removed.